requirements.txt

```diff
@@ -1,3 +1,4 @@
 anymarkup==0.5.0
 lockfile==0.10.2
 jsonpointer==1.10.0
+requests==2.8.1
```

Comment:
To note: We also changed the way that we pull requirements, so "requests" will need to be pulled from yum/apt-get/epel